src/plugins/irc-events/topic.js

```diff
@@ -39,7 +39,7 @@ module.exports = function(irc, network) {
 			type: Msg.Type.TOPIC_SET_BY,
 			mode: chan.getMode(data.nick),
 			nick: data.nick,
-			when: data.when,
+			when: new Date(data.when * 1000),
```

Comment:
Is this `new Date(...)` wrapping necessary since it's [also done at parsing time](https://github.com/thelounge/lounge/pull/167/commits/78e09e8bdc189f64a3dbeb8255b775007fabfe63#diff-9cc146ed670aebf612d9daeff012d09bR3)?